topics/icrc_33_call_canister.md

```diff
@@ -0,0 +1,191 @@
+        - If the status of the call is `replied`:
+            - The relying party retrieves the `reply` blob from the `certificate`'s state tree.
+            - If the `reply` blob or the error information (comprised of the `reject_code`, `reject_message`) is not present, the relying party rejects the response. 
+        - If the status of the call is `rejected`:
+            - The relying party uses the calculated request id to retrieve `reject_code`, `reject_message` from the `certificate`'s state tree.
+            - If the error information (comprised of the `reject_code`, `reject_message`) is not present, the relying party rejects the response.
+        - If the status of the call is `done`:
+            - The relying party now knows that the call was submitted to the IC and executed as requested. However, the result of the call is no longer available.
+    - On error: The relying party may use the information returned to help investigate the causes for the error. The error information is only informative. The relying party must not trust the information returned by the signer and should not use it to make any decisions.
+
+```mermaid
+sequenceDiagram
+    participant RP as Relying Party
+    participant S as Signer
+    participant U as User
+    participant C as Target Canister
+
+    RP ->> S: Request canister call
+    alt Version is not supported
+        S ->> RP: Error response: Version not supported (20101)
+    else Relying party has not been granted the `icrc33_call_canister` permission scope<br>or the request does not comply with scope restrictions
+        S ->> RP: Error response: Permission not granted (30101)
+    else
+        alt Canister supports ICRC-21
+            Note over S,C: Follow the ICRC-21 standard
+        else Canister does not support ICRC-21
+            S ->> U: Display warning and canister call details (canisterId, sender, method, arg)
+            Note over S,U: The warning should inform the user that the canister does not support ICRC-21<br/>The arguments should be decoded, otherwise another warning must be displayed
+        end
+        alt Approved
+            U ->> S: Approve request
+            S ->> C: Submit canister call
+            S ->> S: Wait for the canister call result
+            alt Call successful
+                S ->> U: Display success message
+                S ->> RP: Canister call response
+                RP ->> RP: Validate the certificate
+                RP ->> RP: Retrieve the result
+            else Call failed
+                S ->> U: Display failure message
+                S ->> RP: Error response: Network error (40001) | Unknown error (10001)
+            end
+        else Rejected
+            U ->> S: Reject request
+            S ->> RP: Error response: Action aborted (30201)
+        end
+    end
+```
+
+
+## Scope (according to the [ICRC-25 standard](./icrc_25_signer_interaction_standard.md))
```

Comment:
This section should probably be consistent with ICRC-31 and 32, where it is combined with the `Method`. I like this variant more, though.
But I would suggest moving it upwards, right after the `Method` header.